packages/web/src/fixtures/dev-kit/client.ts

```diff
@@ -118,3 +118,32 @@ export const allocate = async (metricsAddress: string) => {
   }
   return undefined
 }
+
+export const createProperty = async (name: string, symbol: string, author: string) => {
+  const client = newClient()
+  if (process.env.NODE_ENV == 'production' && client) {
+    return client
+      .propertyFactory(await client.registry(addresses.eth.main.registry).propertyFactory())
+      .createProperty(name, symbol, author)
+  } else if (process.env.NODE_ENV == 'development') {
+    console.log('env:', process.env.NODE_ENV, 'return mock value')
+    return 'Dummy:0xd5f3c1bA399E000B1a76210d7dB12bb5eefA8e47'
+  }
+  return undefined
+}
+
+export const marketScheme = async () => {
+  const client = newClient()
+  if (client) {
+    return client.market(await client.registry(addresses.eth.main.registry).marketFactory()).schema()
+  }
+  return []
+}
+
+export const authenticate = async (address: string, args: string[]) => {
+  const client = newClient()
+  if (client) {
+    return client.market(await client.registry(addresses.eth.main.registry).marketFactory()).authenticate(address, args)
```

Comment:
The passed value of `client.market()` is a Market address that will authenticate. That's contained in the URL as /auth/**[market]**/[property]. @k3nt0w

Reply:
I've got a string to type into the placeholder! Thank you very much.